src/MainWindow.cc

```diff
@@ -2369,6 +2372,14 @@ void MainWindow::exceptionCleanup(){
 	if (designActionAutoReload->isChecked()) autoReloadTimer->start();
 }
 
+void MainWindow::UnknownExceptionCleanup(){
+    setCurrentOutput();     // we need to show this error
+    LOG(message_group::Error,Location::NONE,"","Parsing aborted by unknown exception");
+    LOG(message_group::None,Location::NONE,""," ");
+    GuiLocker::unlock();
+    if (designActionAutoReload->isChecked()) autoReloadTimer->start();      // should we stop timers on unknown errors?
```

Comment:
I think this it's pretty innocent to keep it like this, as long as it doesn't keep spewing errors (which it shouldn't). I think we can remove the comment.

Reply:
I looked at the timer code, but couldn't understand how it was used very quickly.
If it is safe, then we can remove the comment.

Reply:
It's a bit of a hack used to discover file changes for the auto-reload UI feature. Ideally this should be rewritten into using a filesystem monitor.